### ISSUE TITLE
"Facebook Comments" do not work for single posts.

### ISSUE THREAD

**Issue by FlorianWolters** [plusjade/jekyll-bootstrap#86]
Hi,

I just discovered that the way "Facebook Comments" are included only works for the entire site and not for every single post.

The file `_includes/JB/comments-providers/facebook` has to be changed to the following to fix this issue:

```
<div id="fb-root"></div>
<script>
    (function(d, s, id) {
        var js, fjs = d.getElementsByTagName(s)[0];
        if (d.getElementById(id)) return;
        js = d.createElement(s); js.id = id;
        js.src = "//connect.facebook.net/en_US/all.js#xfbml=1&appId={{ site.JB.comments.facebook.appid }}";
        fjs.parentNode.insertBefore(js, fjs);
    }(document, 'script', 'facebook-jssdk'));
</script>
<div class="fb-comments" data-href="{{ site.production_url }}{{ page.url }}" data-num-posts="{{ site.JB.comments.facebook.num_posts }}" data-width="{{ site.JB.comments.facebook.width }}" data-colorscheme="{{ site.JB.comments.facebook.colorscheme }}"></div>
```

I only changed `data-href="{{ site.production_url }}"` to `data-href="{{ site.production_url }}{{ page.url }}"`.

**Comment by groundh0g** [comment]
@FlorianWolters's fix [0f60744](https://github.com/FlorianWolters/florianwolters.github.com/commit/0f60744e2203bf6b584775a9b23e68bbe86849ac) needs to be made in the official repo. I won't add all the formatting changes, just the edit to the existing line ...

```
// in file: jekyll-bootstrap/_includes/JB/comments-providers/facebook
<div class="fb-comments" data-href="{{ site.production_url }}" data-num-posts="...
```

... to read as follows.

```
<div class="fb-comments" data-href="{{ site.production_url }}{{ page.url }}" data-num-posts="...
```

**Comment by groundh0g** [comment]
Looks like this is fixed in master. Closing. If you're still having issues, feel free to reopen.
